Adyen.docc/API Reference.md

```diff
@@ -61,6 +61,7 @@ The Adyen DropIn/Components SDK API Reference.
 - ``StoredCardPaymentMethod``
 - ``StoredBCMCPaymentMethod``
 - ``StoredBLIKPaymentMethod``
+- ``StoredACHDirectDebitPaymentMethod``
```

Comment:
Is this related to `CashApp` ?

Reply:
No just I realized it was not there, so I added it.